components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java

```diff
@@ -991,19 +991,22 @@ public class SQLQueries {
 
     public static final String LIST_ALL_TOKENS_IN_TENANT = "SELECT ACCESS_TOKEN, REFRESH_TOKEN, " +
             "TIME_CREATED, REFRESH_TOKEN_TIME_CREATED, VALIDITY_PERIOD, REFRESH_TOKEN_VALIDITY_PERIOD, USER_TYPE, " +
-            "TOKEN_SCOPE, ACCESS_TOKEN_TABLE.TOKEN_ID, AUTHZ_USER, ACCESS_TOKEN_TABLE.USER_DOMAIN, CONSUMER_KEY FROM " +
+            "TOKEN_SCOPE, ACCESS_TOKEN_TABLE.TOKEN_ID, AUTHZ_USER, ACCESS_TOKEN_TABLE.USER_DOMAIN, CONSUMER_KEY, " +
+            "AUTHORIZED_ORGANIZATION FROM " +
             "(SELECT AUTHZ_USER, USER_DOMAIN, CONSUMER_KEY_ID, TOKEN_ID, ACCESS_TOKEN, REFRESH_TOKEN, TIME_CREATED, " +
-            "REFRESH_TOKEN_TIME_CREATED, VALIDITY_PERIOD, REFRESH_TOKEN_VALIDITY_PERIOD, USER_TYPE FROM " +
+            "REFRESH_TOKEN_TIME_CREATED, VALIDITY_PERIOD, REFRESH_TOKEN_VALIDITY_PERIOD, USER_TYPE, " +
+            "AUTHORIZED_ORGANIZATION FROM " +
```

Comment:
Shall we format this?

Reply:
Please check the other places as well.

Reply:
Treated in this manner, as all the below lines will be marked as changed if it was refactored. It will be easy to identify what has changed.